src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Announcing the Meshery Adapter for Open Service Mesh"
+subtitle: ""
+date: 2020-08-20 10:17:05 +0000
+author: Lee Calcote
+thumbnail: ./meshery-open-service.png
+tags:
+ - Community
+ - Projects
+ - News
+published: true
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import conformance from "./conformance-results.png";
+import meshery from "./meshery-open-service.png"
+
+<BlogWrapper>
+
+### Adding another service mesh to the landscape and another adapter to Meshery
+Released on August 5th, 2020 by Microsoft, [Open Service Mesh](https://openservicemesh.io/) (OSM) is a lightweight
+
+<img src={meshery} className="image-right" alt="meshery-adapter-for-open-service-image" />
+
+and [Service Mesh Interace conformant](https://layer5.io/smi) (SMI). Open Service Mesh is a contemporary addition to the [service mesh landscape](https://github.com/layer5io/layer5/blob/master/landscape). Using Envoy as its data plane proxy component and SMI specifications as it's control plane APIs, OSM draws lessons and code from existing service mesh projects, like Linkerd. The Open Service Mesh project has some miles to go as it is one of a growing list of choices available in the service mesh landscape.
+
+First pronounced to be SMI compliant by [Meshery](https://meshery.io/), the service mesh management plane, the first release of OSM supports a myriad of basic
+
+<ul>
+    <li>Securing service to service links</li>
+    <li>Supporting traffic shifting</li>
+    <li>Managing observability for your services</li>
+    <li>Validating and Implementing access control policies</li>
+    <li>Auto addition of applications and services</li>
+</ul>
+
+### Get started with OSM using Meshery
+In Layer5's effort to support our multi-mesh world, our Meshery project provides an effortless way for Kubernetes operators to install, maintain and run service meshes. [Meshery v0.4.3](https://github.com/layer5io/meshery/releases/tag/v0.4.3) includes the [Meshery Adapter for Open Service Mesh](https://github.com/layer5io/meshery-osm), enabling you to quickly provision OSM, run any number of sample applications, manage its performance using [Service Mesh Performance](https://smp-spec.io/) (SMP), validate OSM's compliance to SMI using a suite of conformance tests. Meshery offers configuraiton management with builtin best practice configuration analysis giving you confidence in applying custom configuration to OSM. Meshery's documenation on the [Open Service Mesh integation](https://meshery.layer5.io/docs/service-meshes/adapters/osm) provides a complete walkthrough on how to get set up, install, deploy and configure OSM according to your needs.
+
+<img src={conformance} className="" alt="meshery-smi-conformance-results-image" />
+
+Try Open Service Mesh now, by [getting started with Meshery](https://github.com/layer5io/layer5/blob/master/meshery#getting-started).
+
+-- [Lee Calcote](https://github.com/layer5io/layer5/blob/master/community/members/lee-calcote)
+
```

Comment:
https://github.com/layer5io/layer5/blob/master/community/members/lee-calcote 
The above link is not working. Change it to (https://layer5ng.netlify.app/community/members/lee-calcote)

Reply:
> https://github.com/layer5io/layer5/blob/master/community/members/lee-calcote
> The above link is not working. Change it to (https://layer5ng.netlify.app/community/members/lee-calcote)

Oh thanks, I'll do that asap.